ruff.toml

```diff
@@ -36,9 +35,14 @@ ignore = [
     "E721",
     "E731",
     "E741",
+    "F403",  # `from ... import *` used; unable to detect undefined names
+    "F405",  # may be undefined, or defined from star imports
+    "F821",  # Undefined name
```

Comment:
Checking for undefined names seems pretty useful to me; why did you disable it?

Reply:
It's temporary. Too many changes, and many of them break CI. I'd rather apply them in a dedicated PR. Meanwhile, I'll add a `TODO`.